app/views/org_admin/templates/_show.html.erb

```diff
@@ -10,10 +10,10 @@
   <dd>
     <% if template.published? %>
       <%= _('Published') %>
-    <% elsif template.version == 0 %>
+    <% elsif template.version <= 0 %>
       <%= _('Unpublished')%>
     <% else %>
-      <%= _('You have unpublished changes') %>
+      <%= _('Draft') %>
```

Comment:
- unpublished is just published to false and
- draft is whenever a template version is greater than the published for the same family_id

am I right?